src/preferences/dialog/dlgprefwaveformdlg.ui

```diff
@@ -473,20 +473,36 @@ Select from different types of displays for the waveform, which differ primarily
         <string/>
        </property>
        <property name="text">
-        <string>Beat Grid</string>
+        <string>Beat Grid Opacity</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
-      <widget class="QCheckBox" name="beatGridLinesCheckBox">
+      <widget class="QSlider" name="beatGridAlphaSlider">
+       <property name="maximum">
+        <number>100</number>
+       </property>
+       <property name="value">
+        <number>90</number>
+       </property>
+       <property name="orientation">
+        <enum>Qt::Horizontal</enum>
+       </property>
+      </widget>
+     </item>
+     <item row="4" column="3">
+      <widget class="QSpinBox" name="beatGridAlphaSpinBox">
        <property name="toolTip">
-        <string>Beat Grid</string>
+        <string>Set amount of opacity on beat grid lines.</string>
        </property>
-       <property name="text">
-        <string>Show Beat Markers</string>
+       <property name="maximum">
+        <number>100</number>
+       </property>
+       <property name="value">
+        <number>90</number>
        </property>
```

Comment:
It's not totally clear what the units of this number are. Add this to show a percent sign in the spinbox:
```
<property name="suffix">
 <string notr="true"> %</string>
</property>
```

Reply:
While you're at it, could you also add this for the framerate spinbox:
```
<property name="suffix">
 <string notr="true"> fps</string<
</property>
```